modules/account-lib/src/coin/sol/iface.ts

```diff
@@ -51,7 +51,13 @@ export interface WalletInit {
 
 export interface Transfer {
   type: InstructionBuilderTypes.Transfer;
-  params: { fromAddress: string; toAddress: string; amount: string };
+  params: {
+    fromAddress: string;
+    toAddress: string;
+    amount: string;
+    mint?: string;
+    owner?: string;
```

Comment:
consider extending Transfer to something like TokenTransfer to have required fields mint and owner